src/mlpack/core/math/random.hpp

```diff
@@ -48,6 +48,19 @@ inline void RandomSeed(const size_t seed)
   #endif
 }
 
+/**
+ * Set the random seed to a fixed number.
+ */
+#if (BINDING_TYPE == BINDING_TYPE_TEST)
+inline void SetFixedRandomSeed()
```

Comment:
Not sure about the method name, ``Set`` is rarely used over the codebase. Perhaps ``FixedRandomSeed`` is just fine as well?